src/components/Referrals/ReferralJourney/ReferralJourney.tsx

```diff
@@ -52,24 +70,38 @@ const ReferralJourney = React.memo((props: ReferralJourneyProps) => {
   }, [invitedUsersAccepted])
 
   useEffect(() => {
-    const currentTierIndex = TIERS.findIndex(tier => invitedUsersAccepted < tier.invitesAccepted)
-    const maxTierIndex = currentTierIndex === -1 ? totalSteps : currentTierIndex
+    const currentAnimatedStep = getTiersCompletedCount(invitedUsersAccepted)
```

Comment:
Just to confirm this, is this use effect here to change the `currentAnimatedStep` when it gets loaded?